superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -1183,7 +1184,12 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
                 </StyledInputContainer>
                 <StyledInputContainer>
                   <div className="control-label">
-                    {t('Value')}
+                    {t('Value')}{' '}
+                    <InfoTooltipWithTrigger
+                      tooltip={t(
+                        'Threshold value should be double precision number',
```

Comment:
it doesn't need to be a double right? an integer is still fine. do we even need the tooltip now? Or if you want to add it, you could make it "Threshold can be an integer or a float"

Reply:
double precision number could be integer or float right? 
And, if you type in `3.14159265358979323846264338327950288419716939937510` it will be truncated to `3.141592653589793`. I think it is necessary to let users know there is a limit.

Reply:
ah, i got confused by the fact that the schema called it a `float`, but python calls its doubles floats 🤦  carry on